SPECS/prometheus/prometheus.spec

```diff
@@ -133,6 +135,9 @@ fi
 %doc README.md RELEASE.md documentation
 
 %changelog
+* Wed Nov 06 2024 Nicolas Guibourge <nicolasg@microsoft.com> - 2.37.9-2
+- Patch for CVE-2023-39325 CVE-2023-45288
+
 * Tue Oct 08 2024 Bhagyashri Pathak <bhapathak@microsoft.com> - 2.37.0-16
```

Comment:
This previous changelog entry looks incorrect, can you fix it? should be `2.37.9-1`

Reply:
Here is the last PR which set the changelog version incorrectly - https://github.com/microsoft/azurelinux/commit/f43a558e8d0d7cbc61343109d1b4a57dfba2b3ac#diff-e5fb8a2adb0f25c66b0bd25b3759fbcef2481509b0201f68a948ee5c62f4bcc0R7

Reply:
I've fixed this on your branch